axelrod/strategies/lookerup.py

```diff
@@ -0,0 +1,150 @@
+from axelrod import Player, Actions
+import itertools
+
+C, D = Actions.C, Actions.D
+
+class LookerUp(Player):     
+    """     
+    A strategy that uses a lookup table to
+    decide what to do based on a combination of the last m rounds and the opponent's
```

Comment:
Could you change `last m rounds` to `last m turns` http://axelrod.readthedocs.org/en/latest/reference/glossary.html